### PR TITLE
feat: kubectl support install and uninstall node server

### LINKED ISSUE #247: support exec and logs
**What would you like to be added**:
support exec and logs

**Why is this needed**:
support exec and logs

### PR DESCRIPTION
**Pull request by OrangeBao**
#### What type of PR is this?

/kind feature
#### Which issue(s) does this PR fix?

Fixes #247
#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: pkg/kosmosctl/util/builder.go [hunk 1]
```diff
@@ -179,3 +179,20 @@ func GenerateService(template string, obj interface{}) (*corev1.Service, error)
 	}
 	return o, nil
 }
+
+func GenerattSecret(template string, obj interface{}) (*corev1.Secret, error) {
```
Comment (wuyingjun-lucky) [inline pkg/kosmosctl/util/builder.go]: GenerateSecret

File: pkg/kosmosctl/manifest/manifest_secrets.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package manifest
+
+const (
+	ClusterTreeClusterManagerSecret = `---
```
Comment (ONE7live) [inline pkg/kosmosctl/manifest/manifest_secrets.go:4]: What is the function of this secret?
Would it be better to switch to a time-limited bearer token? @OrangeBao

Comment (OrangeBao) [reply pkg/kosmosctl/manifest/manifest_secrets.go:4]: The HTTPS service will use this certificate, not for clients.

**Pull request merged**